Streetcode/Streetcode.WebApi/appsettings.Local.json

```diff
@@ -7,5 +7,9 @@
     "AllowedHeaders": [ "*" ],
     "AllowedMethods": [ "*" ],
     "PreflightMaxAge": 1
+  },
+  "ReCaptcha": {
+    "SecretKey": "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe",
+    "Url": "https://www.google.com/recaptcha/api/siteverify"
   }
```

Comment:
I'm not sure but that info should probably not be commited to public rep

Reply:
It's ok, don't care)